### PR TITLE
feat: 🎸 Upgrade subql-node and subql-query to latest versions

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Upgrade the SQ images to point to latest versions

### Breaking Changes

NA

### JIRA Link

DA-745

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ services:
       start_period: 30s
 
   graphql-engine:
-    image: onfinality/subql-query:v1.0.0
+    image: onfinality/subql-query:latest
```
Comment (polymath-eric) [inline docker-compose.yml:68]: imo its better to use an explicit version here so it won't break all of a sudden on for someone + we'll be testing with the same versions

File: docker/sq-Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM onfinality/subql-node:v1.13.3
+FROM onfinality/subql-node:latest
```
Comment (polymath-eric) [inline docker/sq-Dockerfile:1]: lets us an explict version here

**Review by polymath-eric — COMMENTED** [review]
lgtm. I think we should use an explicit version for images since `latest` can grab a breaking version and can be annoying to debug.

**Review by polymath-eric — DISMISSED** [review]

**Review by VictorVicente — APPROVED** [review]

**Review by polymath-eric — APPROVED** [review]

**Pull request merged**